common/phish_detector.py

```diff
@@ -191,11 +198,26 @@ def generate_features(self):
                 p.map(run_generator, feature_generators)
                 p.close()
                 p.join()
+            end_time = time.time()
+            min_elapsed, sec_elapsed = int((end_time - start_time) / 60), int((end_time - start_time) % 60)
+            progress_logger.info('Finished feature generation in {} minutes, {} seconds.'.format(min_elapsed, sec_elapsed))
         else:
-            progress_logger.info('Generating features serially...')
+            progress_logger.info('Starting feature generation serially...')
+            start_time = time.time()
+            last_logged_time = start_time
+            dir_count = 0
             for directory in dir_to_generate:
+                dir_count += 1
+                curr_time = time.time()
+                if (curr_time - last_logged_time) > self.logging_interval * 60:
+                    progress_logger.info('Generating feature #{}'.format(dir_count))
```

Comment:
2. A reasonable way would be to truncate the directory so it has just the first 3 characters after `broScripts/output/` and truncate the rest.